docs/documentation/server_admin/topics/user-federation/sssd.adoc

```diff
@@ -18,7 +18,7 @@ image:images/keycloak-sssd-freeipa-integration-overview.png[]
 
 ==== FreeIPA/IdM server
 
-The https://hub.docker.com/r/freeipa/freeipa-server/[FreeIPA Docker image] is available in Docker Hub. To set up the FreeIPA server, see the https://www.freeipa.org/page/Quick_Start_Guide[FreeIPA documentation].
+The https://quay.io/repository/freeipa/freeipa-server?tab=tags/[FreeIPA Container image] is available at https://quay.io/[Quay.io]. To set up the FreeIPA server, see the https://www.freeipa.org/page/Quick_Start_Guide[FreeIPA documentation].
 
 .Procedure
 . Run your FreeIPA server using this command:
```

Comment:
Done, though i'm not sure if this causes any inconsistency in other keycloak documentation (docker vs podman text usage)

Reply:
I did a quick review, and it seems this would be the first time we would have "podman" instead of "docker", so I'm reverting the second commit and keeping only the first one. 

The second change would need some more extended discussion.